### PR TITLE
question 3 layout in place

### PR DESCRIPTION
**Pull request by batteries76**
Smashed through a basic layout for question 3.
https://trello.com/c/DSG4HWLF/210-new-designs-in-question-3-content
This means that the cloud, the heading, and the content are roughly in place and can possibly be used as the template for the other pages that feature these same elements. I think it would be better to arrange this through another layout composition, but also time is against me and I have to move on to other things today.
Also, struggling to work out how to change the `Next` button to the right background colour.
Still, I think all in all, unless there are big things to change here, this one can go through and help out with people taking the future pages that have the same configuration. We can fine tune later.

### CODE REVIEW

**Review by Amutha37 — CHANGES_REQUESTED** [review]
2 areas need changes. The cloud missing features. Buttons need to meet the figma design.

**Comment by hqtan** [comment]
> @batteries76 I can't figure out why, but there's always a scrollbar, even when the content fits comfortably in the viewable height.

@ymloh , I can see the same issue with the scrollbar too; it's always there!

it seems to be [this line of code on L16 on `components/Layout/Layout.jsx`](https://github.com/CodeforAustralia/council-emissions-calculator-spike/pull/36/files#diff-30044812ca947020f04b8092a7616a229e75302aeb620f17587f2aa116ba9c94R16) that's causing the scrollbar to stick around.

File: components/Layout/Layout.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export default function Layout({ children, Progress, isText }) {
         <meta name="description" content="Emissions calculator" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <Flex minHeight="100vh" direction="column">
+      <Flex minHeight="100vh" direction="column" marginBottom={"60px"}>
```
Comment (hqtan) [inline components/Layout/Layout.jsx:16]: the `marginBottom={"60px"}` attribute causes a scrollbar to be ever present on the page, for some reason...
do we need that attribute?

```suggestion
      <Flex minHeight="100vh" direction="column">
```

**Comment by hqtan** [comment]
> 2 areas need changes. The cloud missing features. Buttons need to meet the figma design. Button colour is fixed on question 4. Can get from there.

@Amutha37 , can you be more specific about what feature is missing from the cloud in question 3? I don't see any issues with the cloud icon. I've attached a screenshot of [Question 3](https://council-emissions-calculator-spike-git-fork-98f458-civic-makers.vercel.app/form/Question3) so we can confirm that you and I are seeing the same thing:

 
image

**Comment by batteries76** [comment]
> @batteries76 ,
> 
> I there is an issue with the page, where if you:
> 
> * have provided a number value in the text box, and click 'Next' to proceed to 'Question 4'
> * in 'Question 4', you click 'Back' to go back to 'Question 3'
> * in 'Question 3', your previously entered value is no longer in the text field (i.e. text field is empty), but you are allowed to proceed to 'Question 4' nonetheless. (see screenshot)
>   image
> 
> This can be confusing. Suggestions:
> 
> * Either disable 'Next' button until a value is supplied in text field, or
> * (preferred) retrieve previously entered value, and have value appear in text field

Yes, will have a look into it. Almost tempted to make it a bug card and get the design part through just to have the template for the other pages that people can pull down. But I think I'll work to get that done and push it up.

File: components/Layout/Layout.jsx (hunk 1 above)
Comment (batteries76) [reply components/Layout/Layout.jsx:16]: Yeah. In the mobile screens the buttons are jammed up next to the bottom of the screen.
It looks fine there, but there might be better ways to achieve this.

File: components/LinkButton/LinkButton.jsx [hunk 2]
```diff
@@ -29,9 +30,9 @@ export function BackButton(props) {
 
 export function ContinueButton(props) {
   return (
-    <LinkButton {...props}>
-      <Text>Continue</Text>
-    </LinkButton>
+      <LinkButton {...props}>
+        <Text>Next</Text>
+      </LinkButton>
```
Comment (hqtan) [inline components/LinkButton/LinkButton.jsx:35]: fix indentation
```suggestion
    <LinkButton {...props}>
      <Text>Next</Text>
    </LinkButton>
```

**Comment by hqtan** [comment]
Looking Ace @batteries76 ! :v: 

I left a couple of nitpicks, and a suggestion for improvement; 
but happy to approve it as-is :+1:

**Review by hqtan — APPROVED** [review]
happy dance

**Pull request merged**